charts/atlantis/values.yaml

```diff
@@ -413,3 +413,20 @@ servicemonitor:
 podMonitor:
   enabled: false
   interval: "30s"
+
+# Set the desired Locking DB type
+# lockingDbType: <boltdb|redis>
+
+# Configure Redis Locking DB
+# lockingDbType value must be redis for the config to take effect
+redis: {}
+#  host: redis.host.name
+#  password: myRedisPassword
+#  port: 6379
+#  db: 0
+#  tlsEnabled: false
+#  insecureSkipVerify: false
+
+# If managing secrets outside the chart for the Redis secret, use this variable to reference the secret name
+# redisSecretName: "myRedisSecret"
```

Comment:
Why `redisSecretName` instead of putting `secretName` under `redis` ?

Reply:
`awsSecretName` and `apiSecretName` follow a similar convention.